### PR TITLE
tests: add dsize option signature tests

### PR DESCRIPTION
**Pull request by jmtaylor90**
Signed-off-by: jason taylor <jtfas90@gmail.com>

### CODE REVIEW

File: tests/test-dsize-values/test.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# *** Add configuration here ***
+
```
Comment (jufajardini) [inline tests/test-dsize-values/test.yaml:2]: nit: remove these lines

**Review by jufajardini — CHANGES_REQUESTED** [review]
Aside a nit comment, it looks good to me! :)

File: tests/test-dsize-values/test.yaml (hunk 1 above)
Comment (jmtaylor90) [reply tests/test-dsize-values/test.yaml:2]: oops, will do. thanks for reviewing!

**Comment by jmtaylor90** [comment]
continued in #918

**Pull request closed without merging**